### PR TITLE
Enlarge resource alive timeout

### PR DESCRIPTION
**Pull request by terry-xiaoyu**
Some resources may take very long time to get connected, 5s is not enough in this cases.

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/` dir
- [ ] For EMQX 4.x: `appup` files updated (execute `scripts/update-appup.sh emqx`)
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

### CODE REVIEW

File: apps/emqx_rule_engine/src/emqx_rule_engine.erl [hunk 1]
```diff
@@ -95,6 +95,9 @@
             end
         end()).
 
+-define(GET_RES_ALIVE_TIMEOUT, 60000).
+-define(TEST_RES_PREFIX, "_@test@_:").
```
Comment (zmstone) [inline apps/emqx_rule_engine/src/emqx_rule_engine.erl:99]: Test-> probe
And maybe avoid using @ in the id, not nice for log indexer.
_ instead?

File: apps/emqx_rule_engine/src/emqx_rule_engine.erl [hunk 2]
```diff
@@ -531,12 +534,16 @@ refresh_rule(#rule{id = RuleId, for = Topics, actions = Actions}) ->
...
 refresh_resource_status() ->
     lists:foreach(
-        fun(#resource{id = ResId, type = ResType}) ->
-            case emqx_rule_registry:find_resource_type(ResType) of
-                {ok, #resource_type{on_status = {Mod, OnStatus}}} ->
-                    fetch_resource_status(Mod, OnStatus, ResId);
-                _ -> ok
-            end
+        fun
+            (#resource{id = <<?TEST_RES_PREFIX, _/binary>>}) ->
```
Comment (zmstone) [inline apps/emqx_rule_engine/src/emqx_rule_engine.erl:538]: Add a help function. is_prober

Comment (terry-xiaoyu) [reply apps/emqx_rule_engine/src/emqx_rule_engine.erl]: updated

File: apps/emqx_rule_engine/src/emqx_rule_engine.erl (hunk 1 above)
Comment (terry-xiaoyu) [reply apps/emqx_rule_engine/src/emqx_rule_engine.erl]: changed

**Pull request merged**